airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -74,6 +74,21 @@ def retry_token_expired_handler(**kwargs):
     )
 
 
+class RecordUnnester:
+    def __init__(self, fields: Optional[List[str]] = None):
+        self.fields = fields or []
+
+    def unnest(self, records: Iterable[MutableMapping[str, Any]]) -> Iterable[MutableMapping[str, Any]]:
```

Comment:
this function has quite a bit of nested logic and it would be useful to add a comment explaining exactly what we mean by unnesting

Reply:
Done